src/util/strencodings.cpp

```diff
@@ -1,5 +1,5 @@
 // Copyright (c) 2009-2010 Satoshi Nakamoto
-// Copyright (c) 2009-2022 The Bitcoin Core developers
+// Copyright (c) 2024- The Bitcoin Core developers
```

Comment:
I think you changed this copyright notice in the wrong file. I don't think this one should be changed, only the one in the new file `parse_hex.cpp`.

Reply:
Got it, fixed, thanks